ui/src/message_popup/author/new_question_page.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint no-console: "off" */
```

Comment:
👍  Cool, let's tackle separately.  Can you remove the eslint suppression now though?